### PR TITLE
ANC Changes - Malaria lab

### PR DESCRIPTION
**Pull request by anvmn**
#631 
#632

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-02-12 21-49

**Comment by anvmn** [comment]
Selection_1571

File: client/src/elm/Backend/Measurement/Model.elm [hunk 1]
```diff
@@ -1016,6 +1017,14 @@ type TestResult
     | TestIndeterminate
 
 
+type BloodSmearResult
```
Comment (mariano-dagostino) [inline client/src/elm/Backend/Measurement/Model.elm:1020]: @anvmn Do you think is worth to refactor this to something more specific:

```
type BloodSmearResult = Maybe BloodSmearTestResult

type BloodSmearTestResult = Negative 
                          | Positive NumberOfPlus
```

It seems a bit error prone to have everything inside one type. But up to you.

**Review by mariano-dagostino — APPROVED** [review]

File: client/src/elm/Backend/Measurement/Model.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Backend/Measurement/Model.elm:1020]: @mariano-dagostino 
We list all possible results here, so, no need to refactor.
We have same structure at numerous other places, and there were no issues with it.

**Pull request merged**